odoo/addons/base/models/res_country.py

```diff
@@ -31,7 +31,7 @@
 class ResCountry(models.Model):
     _name = 'res.country'
     _description = 'Country'
-    _order = 'name'
+    _order = 'name, id'
```

Comment:
There is a unique constraint on name.

Reply:
The name is translated, then the name is not unique for the current lang of the user => then the order is non deterministic :smiling_face_with_tear: